src/AspNetCore.Lambda/HttpHandlers.fs

```diff
@@ -275,4 +277,15 @@ let htmlFile (relativeFilePath : string) =
                 ctx
                 |> (setHttpHeader "Content-Type" "text/html"
                 >>= setBodyAsString html)
-        }
\ No newline at end of file
+        }
+
+let razor (engine : RazorLightEngine) (view : string) (model : obj) =
+    let view = engine.Parse(view, model)
+    setHttpHeader "Content-Type" "text/html"
+    >>= setBodyAsString view
+
+let configureRazor (viewsFolder : string)  = 
+    let views = combinePaths (Directory.GetCurrentDirectory()) viewsFolder
+    razor (EngineFactory.CreatePhysical(views))
+
+let defaultRazor () = configureRazor ""
```

Comment:
i'm not creating the engine on each call (i think so), those functions `configure` and `default` returns a partial applied function with the engine, so, in the app you do `let view = razor ()` and now the engine is created once and you can use it wherever `view "foo" model` But, however, hey! I like your approach, uses the built in DI and looks more "aspnet friendly" 😸